getting-started/routing.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Routing"
+weight: 30
+---
+
+# Routing
+
+Routing is an algorithm which defines how chats on the license are distributed between the agents. This document describes chat.io product routing but in the future you will be able to write your own routing algorithm and run it on our platform.
+
+
+### Introduction
+
+In chat.io product it is not possible for a customer to have multiple chats within a single license, the conversation with a license (company) is continous from customer perspective and it is not nessesary to split that into multiple chats. Agents are assigned to this chat automaticly, or manually depends on routing setting.
```

Comment:
In chat.io product it is not possible for a customer to have multiple chats within a single license, the conversation with a license (company) is continous from customer perspective and it is not nessesary to split that into multiple chats. Agents are assigned to this chat automaticly, or manually depends on routing setting. -> In chat.io product it's not possible for a customer to have multiple chats within a single license. The conversation within a license (i.e. a company) is continuous from the customer's perspective and it's not necessary to split it into multiple chats. Agents are assigned to this chat either automatically or manually depending on the routing settings.